src/main/scala/org/embulk/output/fluentd/ColumnVisitor.scala

```diff
@@ -0,0 +1,47 @@
+package org.embulk.output.fluentd
+
+import org.embulk.spi.{
+  Column,
+  PageReader,
+  ColumnVisitor => EmbulkColumnVisitor
+}
+
+case class ColumnVisitor(reader: PageReader) extends EmbulkColumnVisitor {
+  import scala.collection.mutable
+
+  private val record = mutable.Map[String, AnyRef]()
+
+  override def timestampColumn(column: Column): Unit =
+    value(column, reader.getTimestamp).foreach(v => put(column, v.toString))
```

Comment:
It's better to use `TimestampFormatter` here https://github.com/embulk/embulk/blob/master/embulk-core/src/main/java/org/embulk/spi/time/TimestampFormatter.java
Because it makes timestamp values' format configurable.